benchmarks/6.1.0/v2/README.md

```diff
@@ -0,0 +1,3 @@
+# V2
+
+The the newly introduced features/improvements to Identity Server 6.1 base product were being used when getting these performance results. Please refer to the [release note](https://is.docs.wso2.com/en/6.1.0/references/about-this-release/) for further information.
```

Comment:
In this version of Performance results, the tests were executed against the default 6.1.0 base product without enabling or disabling any additional features.

In order to see the performance results which can more comparable with the earlier versions, please refer [the v1 results](url) which disable some of the newly added features/improvements.

Please note that in both cases, [performance tuning parameters](https://is.docs.wso2.com/en/latest/deploy/performance/performance-tuning-recommendations/) has been fine tuned as needed.

Reply:
Fixed with 1c1cd6bce881baf814f28ed1b26616c94e6cc592